tests/core/test_tracker_stores.py

```diff
@@ -398,8 +398,8 @@ def test_db_url_with_query_from_endpoint_config(tmp_path: Path):
         driver: my-driver
         another: query
     """
-    f = tmp_path / "tmp_config_file.yml"
-    f.write_text(endpoint_config)
+    stories_path = tmpdir / "stories.yml"
```

Comment:
Not sure why these modifications were needed? In addition to the code quality failings, the test also fails.

Reply:
I guess it was some post-merge stuff, sorry 🙈
Removed it!